src/common/transformations/tests/transpose_sinking/ts_shape_of_test.cpp

```diff
@@ -0,0 +1,70 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "transformations/transpose_sinking/ts_shape_of.hpp"
+
+#include "common_test_utils/ngraph_test_utils.hpp"
+#include "transformations/transpose_sinking/ts_fuse.hpp"
+
+using namespace ov;
+
+using TSShapeOfForward = TransformationTestsF;
+
+TEST_F(TSShapeOfForward, v0ShapeOf) {
```

Comment:
you can use /home/tikhonov/OpenVINO/openvino/src/common/transformations/tests/transpose_sinking/ts_common_test.cpp
for simple scenarios